### PR TITLE
DATCAP-225: Fix brand and collection broken links

### PR DESCRIPTION
**Pull request by murkas**

### CODE REVIEW

**Review by spacekitcat — APPROVED** [review]

**Review by ajroberts10 — APPROVED** [review]
Looks good, nice fix. Has this been broken for a while? Looks like it would never have worked unless something has changed behind the scenes?

File: src/Controller/Podcast/PodcastController.php [hunk 1]
```diff
@@ -88,7 +88,7 @@ public function __invoke(
         switch ($coreEntity->getType()) {
             case 'collection':
                 $soundsSubscribeUrl = $router->generate('sounds_collection', [
-                    'collectionPid' => $coreEntity->getPid(),
+                    'collectionBrandPid' => $coreEntity->getParent()->getPid(),
```
Comment (ColinBarnwell) [inline src/Controller/Podcast/PodcastController.php:91]: Should this be `getTleo`?

Comment (murkas) [reply src/Controller/Podcast/PodcastController.php]: Now modified as discussed! Thanks! 😄

**Comment by murkas** [comment]
> Looks good, nice fix. Has this been broken for a while? Looks like it would never have worked unless something has changed behind the scenes?

It looks to me as though this may be due to a change to Sounds' routing/redirects that will have been in place previously but have since been removed. Albeit, I wouldn't be able to guess at how long this may have been the case for.

**Review by ColinBarnwell — APPROVED** [review]
👍

**Pull request merged**